manifests/deploy/cluster_role.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:controller:combo
+rules:
+- apiGroups: ["combo.io"]
+  resources: ["*"]
+  verbs: ["*"]
```

Comment:
Nit: Newline.

Reply:
Scope the permissions of the controller. For example, I don't believe that this controller needs to delete either of the combo.io CRs.